src/mockSdk.js

```diff
@@ -14,11 +14,19 @@ export const mockSDKPerson = {
   orgId: 'orgID',
 };
 
-export const mockSDKMediaStreams = {
-  localAudio: new MediaStream(),
-  localVideo: new MediaStream(),
+export const createMockSDKMediaStreams = () => {
+  const localAudio = new MediaStream([{kind: 'audio'}]);
+  const localVideo = new MediaStream([{kind: 'video'}]);
+
+  return {
+    localAudio,
+    localVideo,
+    localAudioVideo: new MediaStream([...localAudio.getTracks(), ...localVideo.getTracks()]),
+  };
 };
 
+const mockSDKMediaStreams = createMockSDKMediaStreams();
```

Comment:
I guess this is no longer needed

Reply:
yes, because each new mock sdk instance needs to have fresh audio & video media streams. (we will set flags on tracks to false and we don't want this to spill from one test to another)